core/app/models/spree/refund.rb

```diff
@@ -29,26 +26,32 @@ def total_amount_reimbursed_for(reimbursement)
       end
     end
 
+    def perform_and_save!
+      perform!
+      save!
```

Comment:
At that point all `perform_and_save!` would do is call `perform!`, as it really doesn't need to return `self` either.

I think we should stick with the first iteration of having to call the public method `perform!` on a refund to perform it, and rely on the Changelog or the RDoc on the method to inform users of the need to explicitly perform their refunds.